From 9aa6122d5a16e67c3efbe1b4877eaec3b1e1f1f6 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sun, 9 Jan 2022 14:09:24 +0000
Subject: [PATCH] GCC: remove constexpr from autofill::LanguageCode()

base::StrongAlias() calls non-constexpr std::string()
constructor. Therefore autofill::LanguageCode() cannot be
constexpr.
---
 components/autofill/core/common/language_code.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/autofill/core/common/language_code.h b/components/autofill/core/common/language_code.h
index d511689..ae6570f 100644
--- a/components/autofill/core/common/language_code.h
+++ b/components/autofill/core/common/language_code.h
@@ -27,7 +27,7 @@ class LanguageCode
   using BaseClass = base::StrongAlias<LanguageCodeTag, std::string>;
 
  public:
-  constexpr LanguageCode() = default;
+  LanguageCode() = default;
   explicit LanguageCode(const char* s) : BaseClass(s) { Check(); }
   explicit LanguageCode(std::string&& s) : BaseClass(std::move(s)) { Check(); }
   explicit LanguageCode(const std::string& s) : BaseClass(s) { Check(); }
-- 
2.34.1

